### PR TITLE
Remove /PATH/TO/POOL.XML and add scsi/mpath pool created by file

### PR DESCRIPTION
**Pull request by meinaLi**
For pool-create XML-file:
1) scsi and mpath pool are created directly with this method. 
2) Other pools have already been covered in test cases.

Signed-off-by: meinaLi <meili@redhat.com>

### CODE REVIEW

**Comment by meinaLi** [comment]
Test Results:
1) avocado run --vt-type libvirt virsh.pool_create.positive_test.from_pre_def_pool.scsi_pool
JOB ID     : 0bd9ea39a901b313e96c58d76b11bd6625d8e371
JOB LOG    : /root/avocado/job-results/job-2019-07-31T03.41-0bd9ea3/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.pool_create.positive_test.from_pre_def_pool.scsi_pool: PASS (32.72 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 36.30 s
2) avocado run --vt-type libvirt virsh.pool_create.positive_test.from_pre_def_pool.mpath_pool
JOB ID     : b5fb43d3984729a62038390f78909ba835753e73
JOB LOG    : /root/avocado/job-results/job-2019-07-31T03.42-b5fb43d/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.pool_create.positive_test.from_pre_def_pool.mpath_pool: PASS (5.87 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 9.71 s

**Comment by meinaLi** [comment]
Has already modified the caselink info.

File: libvirt/tests/src/virsh_cmd/pool/virsh_pool_create.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def run(test, params, env):
     Create a libvirt pool from an XML file. The file could be given by tester or
     generated by dumpxml a pre-defined pool.
     """
-    pool_xml_f = params.get("pool_create_xml_file", "/PATH/TO/POOL.XML")
+    pool_xml_f = params.get("pool_create_xml_file")
```
Comment (chloerh) [inline libvirt/tests/src/virsh_cmd/pool/virsh_pool_create.py]: I think it's better to assign it an default value like '' (empty string)

Comment (meinaLi) [reply libvirt/tests/src/virsh_cmd/pool/virsh_pool_create.py]: Updated.

**Pull request merged**